packages/runtime/test/src/parser/parser.ts

```diff
@@ -4,21 +4,111 @@ import { getProgram } from "../../../src";
 import { RUNTIME_ERRORS } from "../../../src/errors/errors-list";
 import { Interpreter } from "../../../src/interpreter/interpreter";
 import {
-  Add, Addr, Addw, And, AppGlobalDel, AppGlobalGet, AppGlobalGetEx,
-  AppGlobalPut, AppLocalDel, AppLocalGet, AppLocalGetEx, AppLocalPut,
-  AppOptedIn, Arg, Assert, Balance, BitwiseAnd, BitwiseNot, BitwiseOr, BitwiseXor,
-  Branch, BranchIfNotZero, BranchIfZero, Btoi, Byte, Bytec, Callsub,
-  Concat, Cover, Dig, Div, DivModw, Dup, Dup2, EcdsaPkDecompress, EcdsaPkRecover,
+  Add,
+  Addr,
+  Addw,
+  And,
+  AppGlobalDel,
+  AppGlobalGet,
+  AppGlobalGetEx,
+  AppGlobalPut,
+  AppLocalDel,
+  AppLocalGet,
+  AppLocalGetEx,
+  AppLocalPut,
+  AppOptedIn,
+  Arg,
+  Assert,
+  Balance,
+  BitwiseAnd,
+  BitwiseNot,
+  BitwiseOr,
+  BitwiseXor,
+  Branch,
+  BranchIfNotZero,
+  BranchIfZero,
+  Btoi,
+  Byte,
+  Bytec,
+  Callsub,
+  Concat,
+  Cover,
+  Dig,
+  Div,
+  DivModw,
+  Dup,
+  Dup2,
+  EcdsaPkDecompress,
+  EcdsaPkRecover,
   EcdsaVerify,
-  Ed25519verify, EqualTo, Err, Exp, Expw,
-  Extract, Extract3, ExtractUint16, ExtractUint32, ExtractUint64, Gaid, Gaids,
-  GetAssetDef, GetAssetHolding, GetBit, GetByte, Gload, Gloads, Global, GreaterThan,
-  GreaterThanEqualTo, Gtxn, Gtxna, Gtxns, Gtxnsa, Int, Intc, Itob, ITxnBegin, ITxnField,
-  ITxnSubmit, Keccak256, Label, Len, LessThan,
-  LessThanEqualTo, Load, Loads, MinBalance, Mod, Mul, Mulw, Not, NotEqualTo,
-  Or, Pop, Pragma, PushBytes, PushInt, Retsub,
-  Return, Select, SetBit, SetByte, Sha256, Sha512_256, Shl, Shr, Sqrt,
-  Store, Stores, Sub, Substring, Substring3, Swap, Txn, Txna, Uncover
+  Ed25519verify,
+  EqualTo,
+  Err,
+  Exp,
+  Expw,
+  Extract,
+  Extract3,
+  ExtractUint16,
+  ExtractUint32,
+  ExtractUint64,
+  Gaid,
+  Gaids,
+  GetAssetDef,
+  GetAssetHolding,
+  GetBit,
+  GetByte,
+  Gload,
+  Gloads,
+  Global,
+  GreaterThan,
+  GreaterThanEqualTo,
+  Gtxn,
+  Gtxna,
+  Gtxns,
+  Gtxnsa,
+  Int,
+  Intc,
+  Itob,
+  ITxnBegin,
+  ITxnField,
+  ITxnSubmit,
+  Keccak256,
+  Label,
+  Len,
+  LessThan,
+  LessThanEqualTo,
+  Load,
+  Loads,
+  MinBalance,
+  Mod,
+  Mul,
+  Mulw,
+  Not,
+  NotEqualTo,
+  Or,
+  Pop,
+  Pragma,
+  PushBytes,
+  PushInt,
+  Retsub,
+  Return,
+  Select,
+  SetBit,
+  SetByte,
+  Sha256,
+  Sha512_256,
+  Shl,
+  Shr,
+  Sqrt,
+  Store,
+  Stores,
+  Sub,
+  Substring,
+  Substring3,
+  Swap,
+  Txn,
+  Txna,
+  Uncover
```

Comment:
did your editor do this?

Reply:
I think I use auto format tool in vs code and it changed the code like this.